test/unit/vl-map-layer.test.html

```diff
@@ -0,0 +1,252 @@
+<!doctype html>
```

Comment:
Het is voor mij persoonlijk niet beter dan wat het voorheen was, omdat als er een test faalt je ook minder duidelijk gaat zien bij welke layer hij faalt en ik denk niet dat je echt veel spaart in termen van regels code?

Maar da's mijn mening, en als niemand hier dezelfde opmerking over heeft is het voor mij prima :)

Reply:
Ik merge al, maar zal opmerking voorlopig nog unresolved laten staan.